jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -1297,7 +1307,14 @@ static void initSecurityManager(ClassLoader applicationClassLoader) {
 				throw new Error(Msg.getString("K0631", e.toString()), e); //$NON-NLS-1$
 			}
 		}
+		/*[ENDIF] JAVA_SPEC_VERSION >= 24 */
 	}
+	/*[IF JAVA_SPEC_VERSION >= 24]*/
+	if (throwErrorOnInit) {
+		/*[MSG "K0637", "A command line option has attempted to allow or enable the Security Manager. Enabling a Security Manager is not supported."]*/
+		throw new Error(Msg.getString("K0637")); //$NON-NLS-1$
+	}
+	/*[ENDIF] JAVA_SPEC_VERSION >= 24 */
```

Comment:
I think `throwErrorOnInit` isn't needed, the `Error` can be moved around Line 1278 whenever `java.security.manager` is detected.

Reply:
The message should be prefixed with "Error: "; see [SecurityManagerWarnings.java](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/test/jdk/java/lang/System/SecurityManagerWarnings.java#L91).

Reply:
That is from `java.lang.Error`. The example output from https://openjdk.org/jeps/486 is:
```
java.lang.Error: A command line option has attempted to allow or enable the Security Manager. Enabling a Security Manager is not supported.
        at java.lang.System.initPhase3(java.base@24/System.java:2067)
```